src/main/java/nl/cherement/jak/controller/UserController.java

```diff
@@ -23,32 +20,25 @@
         return userService.findAllShortened();
     }
 
-    @GetMapping("/me")
-    public UserEntity findUser(Principal principal) {
```

Comment:
waarom gaat deze method weg?

Reply:
Omdat hij onnodig is. Hij wordt niet gebruikt in de frontend en zie niet echt waarvoor hij wel gebruikt zou kunnen worden.